wp-content/themes/csisjti/assets/_scss/components/_filters-modal.scss

```diff
@@ -311,14 +325,19 @@ $facet: '.facetwp-facet';
   /*----------  Facet: Additional Dropdown Styling  ----------*/
 
   .facetwp-type-fselect {
-    max-width: 66%;
+    // max-width: 66%;
```

Comment:
Sorry for not being clear. We want to set the max-width to 66% once the screen gets larger. Eg. medium and above.